src/context.rs

```diff
@@ -1523,6 +1523,74 @@ impl Context {
             Err(ret)
         }
     }
+
+    /// Make transient objects persistent or remove persistent objects
+    pub fn evict_control(
+        &mut self,
+        auth: ESYS_TR,
+        handle: ESYS_TR,
```

Comment:
Check the handle types that we have and see any of the specific handle types are applicable otherwise use the ObjectHandle type.